src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -880,11 +880,8 @@ namespace winrt::TerminalApp::implementation
 
         // Build the commandline to pass to wt for this set of NewTerminalArgs
         // `-w -1` will ensure a new window is created.
-        winrt::hstring cmdline{
-            fmt::format(L"-w -1 new-tab {}",
-                        newTerminalArgs ? newTerminalArgs.ToCommandline().c_str() :
-                                          L"")
-        };
+        const auto newTabCommandline = newTerminalArgs ? newTerminalArgs.ToCommandline() : winrt::hstring{};
+        const auto cmdline = fmt::format(FMT_COMPILE(L"-w -1 new-tab {}"), newTabCommandline);

```

Comment:
I changed all `fmt::format` calls to use `FMT_COMPILE`.
I'm not entirely sure why, but by using `FMT_COMPILE` a bunch of compile errors went away.